### ISSUE TITLE
This software is not open source.

### ISSUE THREAD

**Issue by kfogel** [sequence-so/sequence#8]
This software calls itself "open source", but it is not.  It's under the [Apache 2.0 license + "Commons Clause"](https://github.com/sequence-so/sequence/blob/master/LICENSE.md).  The Commons Clause makes it not open source.

Possible solutions: either switch to a truly open source license (Apache 2.0 alone would be open source), or don't call the software "open source".

**Comment by helsont** [comment]
@kfogel you are correct! We will be updating the copy to reflect its current license. For now we are going to stick to Apache 2.0 + Commons Clause until we hear that it's too restrictive for valid use cases.

**Comment by kfogel** [comment]
Thanks for the fast response, @helsont, and for the upcoming update.
